.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
+  "parserOptions": {
```

Comment:
Actually, the SDK needs to be on ES5, because it's supposed to be isomorphic and compatible with the browser and Node. In fact, we should probably add a check to make sure no one uses ES6 here :)